com.github.gmg137.netease-cloud-music-gtk.yml

```diff
@@ -0,0 +1,46 @@
+app-id: com.github.gmg137.netease-cloud-music-gtk
```

Comment:
`io.github` is usually preferred, but not a blocker.

Reply:
I got this id from `dbus-monitor`